compliance/cis_k8s/rules/cis_1_2_14/data.yaml

```diff
@@ -1,36 +1,36 @@
 metadata:
-  id: 554a999e-c11a-5c1c-aa53-2badd123c122
-  name: Ensure that the admission control plugin NamespaceLifecycle is set
-    (Automated)
+  id: cf6ebe82-9367-5b8e-a117-c81ab11ba7d4
```

Comment:
I don't mind now, but we should avoid changing ids of rules once we are out.
worth discussing it as we go forward about the id strategy in the tool

Reply:
maybe the right thing is to change the ids for all

Reply:
did it with the help of @ofiriro3 and the script of @jeniawhite

Reply:
i'll set up a discussion for this. not critical for this pr (srry)